enrolments/views.py

```diff
@@ -14,5 +20,15 @@ class SessionViewSet(
     http_method_names = [
         "get",
     ]
-
     permission_classes = [permissions.IsAuthenticated]
+
+    @action(
+        methods=["get"],
+        detail=True,
+        permission_classes=[permissions.IsAuthenticated],
+        url_path="classes",
+        url_name="classes",
+    )
+    def get_classes(self, request, pk=None):
+        classes = Session.objects.filter(id=pk).first().classes
```

Comment:
using `.get()` might be a bit cleaner here, since it'll throw appropriate errors if there isn't exactly 1 session that matches! ([docs](https://docs.djangoproject.com/en/3.1/topics/db/queries/#retrieving-a-single-object-with-get))

Reply:
good to know, updated!